src/jsonapi-datastore.js

```diff
@@ -11,10 +11,68 @@
     this.id = id;
     this._type = type;
     this._attributes = [];
+    this._dependents = [];
     this._relationships = [];
   }
 
   /**
+   * Add a dependent to a model.
+   * @method _addDependence
+   * @param {string} type The type of the dependent model.
+   * @param {string} id The id of the dependent model.
+   * @param {string} key The name of the relation found on the dependent model.
+    */
+  _addDependence(type, id, key) {
```

Comment:
If we do as suggested [here](https://github.com/beauby/jsonapi-datastore/pull/28/files#r56110661), we could simply rewrite this as `_removeDependence(type, id, key); this._dependents.push({id: id, type: type, relation: key});`.